Gopkg.lock

```diff
@@ -465,6 +465,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "3629212a518505ac90fb570403f2993c2b16257cff30ecc50363cfadd1c686e4"
+  inputs-digest = "db30bb7a0e5c03ea865378d78c65b2cadbeea1efd3e96f68afae43ebc11cc1a2"
```

Comment:
Why is this here?

Reply:
I vaguely remember that this changed after doing dep ensure, so I added it.